From 26c90728c3a37a6b1862a29d6dfbdd4f1cdfc72b Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 10 Aug 2021 12:39:45 +0200
Subject: [PATCH 010/103] linux: Add test for touchpads being tagged as mice

udev adds both tags to touchpads, so replicate that behaviour.
---
 src/linux/integration-test | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/linux/integration-test b/src/linux/integration-test
index 128477c..211f445 100755
--- a/src/linux/integration-test
+++ b/src/linux/integration-test
@@ -1370,7 +1370,7 @@ class Tests(dbusmock.DBusTestCase):
             'input',
             '/devices/pci0000:00/0000:00:14.0/usb3/3-10/3-10:1.2/0003:046D:C52B.0009/0003:046D:4101.000A/input/input22',
             parent,
-            [], ['DEVNAME', 'input/mouse3', 'ID_INPUT_TOUCHPAD', '1'])
+            [], ['DEVNAME', 'input/mouse3', 'ID_INPUT_TOUCHPAD', '1', 'ID_INPUT_MOUSE', '1'])
         self.testbed.uevent(batt_dev, 'change')
 
         time.sleep(0.5)
@@ -1389,7 +1389,7 @@ class Tests(dbusmock.DBusTestCase):
             'input',
             '/devices/pci0000:00/0000:00:14.0/usb3/3-10/3-10:1.2/0003:046D:C52B.0009/0003:046D:4101.000A/input/input22',
             parent,
-            [], ['DEVNAME', 'input/mouse3', 'ID_INPUT_TOUCHPAD', '1'])
+            [], ['DEVNAME', 'input/mouse3', 'ID_INPUT_TOUCHPAD', '1', 'ID_INPUT_MOUSE', '1'])
 
         dev = self.testbed.add_device(
             'power_supply',
-- 
2.35.1

